### ISSUE TITLE
Cut/Copy/Paste popup overlaps text selection handles in TextFormField

### ISSUE THREAD

**Issue by vlidholt** [flutter/flutter#18112]
When trying to select text, the cut/copy/paste popup is too close to selection handles, making it almost impossible to change the text selection. See attached image.

![screen shot 2018-06-01 at 12 03 49 pm](https://user-images.githubusercontent.com/1539812/40835577-1a03c610-6594-11e8-89d8-1428e8ced076.png)

```
[✓] Flutter (Channel master, v0.5.2-pre.18, on Mac OS X 10.13.4 17E202, locale en-US)
    • Flutter version 0.5.2-pre.18 at /Users/vik/Projects/flutter
    • Framework revision 758711c85d (8 hours ago), 2018-05-31 10:33:15 +0200
    • Engine revision d174c4ff01
    • Dart version 2.0.0-dev.59.0.flutter-ff815d05a5

[!] Android toolchain - develop for Android devices (Android SDK 27.0.3)
    • Android SDK at /Users/vik/Library/Android/sdk
    • Android NDK location not configured (optional; useful for native profiling support)
    • Platform android-27, build-tools 27.0.3
    • Java binary at: /Applications/Android Studio.app/Contents/jre/jdk/Contents/Home/bin/java
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-915-b08)
    ✗ Android license status unknown.

[✓] iOS toolchain - develop for iOS devices (Xcode 9.3.1)
    • Xcode at /Applications/Xcode.app/Contents/Developer
    • Xcode 9.3.1, Build version 9E501
    • ios-deploy 1.9.2
    • CocoaPods version 1.5.3

[✓] Android Studio (version 3.0)
    • Android Studio at /Applications/Android Studio.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    • Dart plugin version 171.4424
    • Java version OpenJDK Runtime Environment (build 1.8.0_152-release-915-b08)

[✓] Android Studio (version 2.3)
    • Android Studio at /Applications/Android Studio - old.app/Contents
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
    • Java version OpenJDK Runtime Environment (build 1.8.0_112-release-b06)

[✓] IntelliJ IDEA Community Edition (version 2017.2.5)
    • IntelliJ at /Applications/IntelliJ IDEA CE.app
    • Flutter plugin version 20.0.2
    • Dart plugin version 172.4343.25

[✓] Connected devices (1 available)
    • iPhone 6 • 5DE7BCCC-8793-413A-9924-3D413DE12FCD • ios • iOS 9.0 (simulator)

! Doctor found issues in 1 category.
```

**Comment by vlidholt** [comment]
Also:
- There is a thin line between the body of the popup-bubble and the triangle/arrow.
- The triangle/arrow should point at the selected text or towards the text cursor (compare to native text selection).

**Comment by zoechi** [comment]
Related to #3945

**Comment by xster** [comment]
I'll not dupe this into #3945 since the technical fix is slightly orthogonal and easier.

**Comment by LongCatIsLooong** [comment]
@xster  This issue seems to have already been fixed on master(37f8020b4ccac57d4f73dffadc3e06554f633bfd)
![2019-03-25_319x130](https://user-images.githubusercontent.com/31859944/54952319-f296aa80-4f02-11e9-94c2-5498254b0b71.png)

**Comment by xster** [comment]
sounds good, let's close it.
